src/dini/headers.js

```diff
@@ -136,15 +135,6 @@ export function run(conf) {
       pub("error", "All alternate formats must have a uri and a label.");
     }
   });
-  conf.multipleAlternates =
-    conf.alternateFormats && conf.alternateFormats.length > 1;
-  conf.alternatesHTML =
-    conf.alternateFormats &&
-    htmlJoinAnd(conf.alternateFormats, alt => {
-      const lang = alt.hasOwnProperty("lang") && alt.lang ? alt.lang : null;
-      const type = alt.hasOwnProperty("type") && alt.type ? alt.type : null;
-      return hyperHTML`<a rel='alternate' href='${alt.uri}' hreflang='${lang}' type='${type}'>${alt.label}</a>`;
-    });
```

Comment:
Is this a breaking change to DINI?

Reply:
I don't think so, dini templates never use them.